packages/typedoc-plugins/LICENSE.md

```diff
@@ -0,0 +1,39 @@
+Software License Agreement
+==========================
+
+@ckeditor/typedoc-plugins – Various TypeDoc plugins developed by the CKEditor 5 team.
+Copyright (c) 2003-2022, CKSource – Holding sp. z o.o. All rights reserved.
+
+Licensed under the terms of the MIT License (see Appendix A):
+
+	http://en.wikipedia.org/wiki/MIT_License
+
+Sources of Intellectual Property Included in CKEditor
+-----------------------------------------------------
+
+Where not otherwise indicated, all @ckeditor/typedoc-plugins content is authored by CKSource engineers and consists of CKSource-owned intellectual property. In some specific instances, @ckeditor/typedoc-plugins will incorporate work done by developers outside of CKSource with their express permission.
+
+Appendix A: The MIT License
+---------------------------
+
+The MIT License (MIT)
+
+Copyright (c) 2003-2022, CKSource – Holding sp. z o.o.
```

Comment:
```suggestion
Copyright (c) 2003-2022, CKSource Holding sp. z o.o.
```

Reply:
I also found 4 more occurrences of this problem. I fixed all of them.